src/store.js

```diff
@@ -44,7 +44,19 @@ export const store = reactive({
     template: '',
     include_test: false,
     output_dir: './logs',
-    log_every_iters: 10
+    log_every_iters: 10,
+    dist: 'torchrun',
+    nproc_per_node: 2,
+    nnodes: 1,
+    master_addr: '127.0.0.1',
```

Comment:
Are you sure you need to add this configs by default ?

Reply:
Yes, when we click on the Training tab then these are the defaults that are set int the `config.json` but as I understand, for these to actually affect the `config.yaml` it has other if conditions in place such as `if (it.dist === 'spawn')`

Reply:
If we check `metadata.json` then we can see the default values that will be set for the variables when clicked on each of the tabs...

I think `dist: 'torchrun'` isn't a default requirement here, will modify that.